src/Axiom/Set.agda

```diff
@@ -167,9 +167,18 @@ record Theory {ℓ} : Type (sucˡ ℓ) where
   ∈-unions : {a : A} {U : Set (Set A)} → (∃[ T ] T ∈ U × a ∈ T) ⇔ a ∈ proj₁ (unions U)
   ∈-unions = proj₂ $ unions _
 
+  fromList-finite : ∀ {l : List A} → finite (fromList l)
+  fromList-finite {l = l} = l , mk⇔ (from ∈-fromList) (to ∈-fromList)
+
   ∅ : Set A
   ∅ = fromList []
 
+  ¬a∈[] : ∀ {a : A} → a ∈ˡ [] → ⊥
```

Comment:
These are just useful to have around (so I don't have to figure out how to prove them again next time).